kratos/geometries/quadrature_point_geometry.h

```diff
@@ -280,6 +280,20 @@ class QuadraturePointGeometry
         return point;
     }
 
+    ///@}
+    ///@name Mathematical Informations
+    ///@{
+
+    /// Returns the polynomial degree of the parent geometry
+    SizeType PolynomialDegree(IndexType LocalDirectionIndex) const override
+    {
+        KRATOS_DEBUG_ERROR_IF(mpGeometryParent == nullptr)
+            << "Trying to call PolynomialDegree(LocalDirectionIndex) from quadrature point. "
+            << "Pointer to parent is not assigned." << std::endl;
```

Comment:
```suggestion
        KRATOS_DEBUG_ERROR_IF_NOT(mpGeometryParent)
            << "Trying to call PolynomialDegree(LocalDirectionIndex) from quadrature point. "
            << "Pointer to parent is not assigned." << std::endl;
```

Reply:
are you sure that works?

Reply:
[yes](https://stackoverflow.com/a/17772117)

Reply:
a yes with a link ❤️